src/app/game/settings/page.tsx

```diff
@@ -0,0 +1,117 @@
+import NavigationFooter from "@/features/game/NavigationFooter";
+import Link from "next/link";
+import { Diamond,
+         Mail,
+         KeyRound,
+         LogOut,
+         Trash2,
+         Gamepad2,
+         Info,
+         FileText,
+         ChevronRight }
+         from 'lucide-react';
+
+export default function GamePage() {
+  const pageTitle = "設定";
+
+  return (
+    <div className="relative h-full">
+      <h1 className="fixed inset-x-0 top-0 bg-[#0094f4] p-4 pt-7 text-center text-3xl text-white">
+        {pageTitle}
+      </h1>
+
+    <div className="mx-auto justify-center pb-24 pt-28">
+        <div className="mx-auto mb-5 flex w-[90%] items-center">
+            <Diamond width={30} height={30} className="mr-5"/>
+
+            <span className="text-3xl">ユーザー設定</span>
+        </div>
+
+        <div className="mx-auto w-3/4">
+            <div className="mb-5 flex items-center">
+                <Mail width={30} height={30} className="mr-5"/>
+
+                <Link href={`/game/settings/account/change/email`}>
+                    <span className="text-2xl">メールアドレス変更</span>
+                </Link>
+
+                <ChevronRight width={30} height={30} className="ml-auto"/>
```

Comment:
アイコンと右矢印までリンクに入れちゃってもいいかもと思いましたがいかがでしょうか？
こちらは自分も判断つき兼ねたので、このままでも大丈夫です！
デプロイ語のUIの動作確認時に使いにくかったら修正すればよいため。
```suggestion
                <Link href={`/game/settings/account/change/email`} className="flex w-full items-center">
                    <Mail width={30} height={30} className="mr-5"/>
                    <span className="text-2xl">メールアドレス変更</span>
                    <ChevronRight width={30} height={30} className="ml-auto"/>
                </Link>
```